luci-app-ssr-plus/Makefile

```
@@ -1,15 +1,16 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=luci-app-ssr-plus
-PKG_VERSION:=187
-PKG_RELEASE:=2
+PKG_VERSION:=188
+PKG_RELEASE:=3
```

```suggestion
PKG_RELEASE:=1
```
当 `PKG_VERSION` 更新时，`PKG_RELEASE` 改为 1

---

Fixed!